lib/rules/no-typos.js

```diff
@@ -114,6 +145,20 @@ module.exports = {
     }
 
     return {
+      ImportDeclaration: function(node) {
+        if (node.source && node.source.value === 'prop-types') { // import PropType from "prop-types"
+          propTypesPackageName = node.specifiers[0].local.name;
+        } else if (node.source && node.source.value === 'react') { // import { PropTypes } from "react"
+          reactPackageName = node.specifiers[0].local.name;
+
+          propTypesPackageName = node.specifiers.length >= 1 ? (
+            node.specifiers
+              .filter(specifier => specifier.imported && specifier.imported.name === 'PropTypes')
+              .map(specifier => specifier.local.name)
+          )[0] : null;
```

Comment:
should this use `.find` instead of `.filter`?

Reply:
Sure; it makes this code clearer. Updated.